packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -61,6 +62,13 @@ import rightClickMenuReducer, {
 } from './reducer';
 import { getDefaultPropertiesForType, getElementStyles } from './utils';
 
+const UNDO_HELP_TEXT = sprintf(
+  /* translators: %s: Command or Control key on keyboard */
+  __('Press %s Z to undo the last change', 'web-stories'),
+  getKeyForOS('mod'),
+  'Z'
+);
```

Comment:
I'd move the whole key combo out of the translatable string and also use `prettifyShortcut`, e.g. something like

```suggestion
const UNDO_HELP_TEXT = sprintf(
  /* translators: %s: Ctrl/Cmd + Z keyboard shortcut */
  __('Press %s to undo the last change', 'web-stories'),
  prettifyShortcut('mod+z')
);
```

Also that second `Z` arg was not doing anything

Could probably

Reply:
Ditto for the instance in `useQuickActions.js`